docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+blah blah some intro
+
+## Prerequisites
+
+Before you begin, understand that this article assumes that you are using Windows and Visual Studio.
```

Comment:
Yeah I'm going to have to think about how to reword this.  The largest audience here will be VS and Windows users on .NET Framework, but this could be useful info for people coming from Mono and MonoDevelop.